apps/lambda/README.md

```diff
@@ -3,15 +3,31 @@ This project uses [`aws-serverless-java-container`](https://github.com/awslabs/a
 
 The starter project defines a number of resources `/`, `/resources/greeting`, and `/resources/names` that can accept `GET` requests.
 
-The project folder also includes a `sam.yaml` file. You can use this [SAM](https://github.com/awslabs/serverless-application-model) file to deploy the project to AWS Lambda and Amazon API Gateway or test in local with [SAM Local](https://github.com/awslabs/aws-sam-local). 
+The project folder also includes three Microservices Serverless Application Model templates `GreetingSam.yaml`, `NameSam.yaml` and `WebApp.yaml` . You can use theses [SAM](https://github.com/awslabs/serverless-application-model) files to deploy the project to AWS Lambda and Amazon API Gateway or test in local with [SAM Local](https://github.com/awslabs/aws-sam-local). 
+
+## Prepare Microservice Source Code
+
+The example application consists of there Microservices, they are Greeting Service, Name Service and WebApp Service. Each service is developed as an self-contained module. WebApp service is responsible for frontend service, Greeting and Name service are responsbile for generating response to partials of to the client.
+
+To start with, please use the following commands to clone these services source code into the workspace
+
+Clone Greeting Service Source Code
+
+```
+cd ../ && mkdir -p services && cd ../services/
```

Comment:
These instructions would not be required with the updated `CodeUri`

Reply:
moved lambda codeuri path change into pipeline